hack/verify-flags/known-flags.txt

```diff
@@ -189,6 +189,8 @@ experimental-allowed-unsafe-sysctls
 experimental-bootstrap-kubeconfig
 experimental-keystone-url
 experimental-mounter-path
+experimental-mounter-rootfs-path
```

Comment:
I don't know where it comes from. Seems rebase from previous commits. I did not add this. Quite weird.

Already removed this. Fixed.